site/content/overview/resource-validation.md

```diff
@@ -131,28 +75,21 @@ kubectl describe gateway some-gateway
 Events:
   Type     Reason    Age   From                            Message
   ----     ------    ----  ----                            -------
-  Warning  Rejected  6s    nginx-gateway-fabric-nginx  the resource failed webhook validation, however the Gateway API webhook failed to reject it with the error; make sure the webhook is installed and running correctly; validation error: spec.listeners[1].hostname: Forbidden: should be empty for protocol TCP; NGF will delete any existing NGINX configuration that corresponds to the resource
```

Comment:
This also requires a basic code change. `internal/mode/static/state/store.go` line 289 is where that NGF string is actually created.

Reply:
reverted this change

Reply:
Why revert? Don't we want to avoid acronyms?

Reply:
as per our conversation, modified to NGINX Management Suite in doc and go file